### PR TITLE
fix: typos, should always include `JSON Schema`, not only `JSON`

### PR DESCRIPTION
**Pull request by jeremyfiel**
**What kind of change does this PR introduce?**

fix typos for the naming `JSON Schema`

**Issue Number:**

-  Closes #___ 
-  Related to #___ 
-  Others? 

**Screenshots/videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

### CODE REVIEW

**Review by DhairyaMajmudar — APPROVED** [review]
LGTM, Thank you so much @jeremyfiel for the PR

File: pages/ambassadors/index.page.tsx [hunk 1]
```diff
@@ -68,10 +68,10 @@ export default function communityPages() {
...
           >
             <h1 className='text-h3 font-bold text-gray-900 dark:text-white'>
-              JSON Ambassador Contributions
+              JSON Schema Ambassador Contributions
             </h1>
             <p className='mt-5 text-lg text-gray-700 dark:text-slate-100'>
-              JSON Ambassadors are passionate about JSON Schema. They share
+              Ambassadors are passionate about JSON Schema. They share
               their interest, expertise, and excitement within their communities
               to help others build better software.
```
Comment (benjagm) [inline pages/ambassadors/index.page.tsx:76]:
```suggestion
              Ambassadors are passionate about JSON Schema. They share their
              interest, expertise, and excitement within their communities to
              help others build better software.
```

**Review by benjagm — CHANGES_REQUESTED** [review]
Thanks a lot for this PR Jeremy!!  Much appreciated. 

Can you please commit my suggestion? We have a process checking the formatting and this suggestion contains the required change.

File: pages/ambassadors/index.page.tsx (hunk 1 above)
Comment (jeremyfiel) [reply pages/ambassadors/index.page.tsx:76]: My changes didn't touch this . Why is it failing on my pr?

Comment (benjagm) [reply pages/ambassadors/index.page.tsx:76]: > My changes didn't touch this . Why is it failing on my pr?

You removed JSON from the sentence, which altered the formatting and now the linter is failing.

**Review by benjagm — APPROVED** [review]
Thanks a lot for this PR

**Pull request merged**